src/calls-cost.js

```diff
@@ -39,6 +39,65 @@
  * @returns {CallsResponse}  - Processed information
 */
 
-function callsCost(calls) { }
+function callsCost(calls) {
```

Comment:
test for this function is failing, did you run the `yarn test` command to check it?
also, if you see in the image there are more than 2 decimals, for this scenario I'd recommend  to handle precision

![image](https://user-images.githubusercontent.com/5667884/150359339-d204d13a-fca2-410c-b542-ea34df72a234.png)